tests/conftest.py

```diff
@@ -11,7 +11,7 @@
 from time_tracker_api import create_app
 from time_tracker_api.database import init_sql
 from time_tracker_api.security import get_or_generate_dev_secret_key
-from time_tracker_api.time_entries.time_entries_model import (
```

Comment:
@josepato87 if it is an unused line, I recommend removing the commented lines

Reply:
As we discuss with @Angeluz-07 we refactor the code because time_entries_model had multiple responsibilities.